feeds/cache.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Cache
+parent: Feeds
+nav_order: 3
+---
+
+# Central Feed Cache
+
+Cache handling for feeds specified by `<PackageSource>` in Central Feed Management is separate from feeds specified in NuGet.config.
+
+## Cache Organization
+
+In order to enforce the security rules of [Feed Scoping](/feeds/grouping-and-source-pinning/#feed-scoping), packages downloaded using Central Feed Management will be stored in a separate cache.
+
+For projects with Central Feed Management enabled, the global package cache will not be utilized.
+
+The Central Feed Management cache organizes a sub-cache for every feed.
+
+#### Cache Storage Locations
+```
+On Linux:
+~/.nuget/packagesource/<feed URL hash>/<packagename>
+
+On Windows:
+%userprofile%\.nuget\packagesource\<feed URL hash>\<packagename>
+```
+
+Here's an example of how a package may be stored:
+```
+On Linux:
+~/.nuget/packagesource/c48fd7be919f1d622a02ea44ccb5d119/newtonsoft.json/12.0.3/
+
+On Windows:
+%userprofile%\.nuget\packagesource\c48fd7be919f1d622a02ea44ccb5d119\newtonsoft.json\12.0.3\
```

Comment:
Please keep path length limitations on Windows into account.

Reply:
This is an excellent point.

From what I can tell, it is now [much easier](https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation#enable-long-paths-in-windows-10-version-1607-and-later) to enable NTFS long file name support for long path names. If we detect that long path names are not enabled on a machine, I believe that we should be able to prompt the user (on a development machine) or automatically update the registry key (in a build environment). In many cases, this will already be enabled.

Since development and build machines are usually running an updated version of Windows 10 client, Windows Server, or Linux, this should solve the path length limitations for a very high percentage of users.

I would be interested to hear more feedback on this, but it seems like enabling NTFS long file names once to improve cache organization and security is a worthwhile tradeoff for the more experienced users who need Central NuGet Management.